### PR TITLE
SM: Fix Trick Names and Descriptions

### PR DESCRIPTION
**Pull request by Logarhythmic**
Removed duplicate entry and updated descriptions.

### CODE REVIEW

File: randovania/games/super_metroid/json_data/header.json [hunk 1]
```diff
@@ -265,7 +265,7 @@
             },
             "IBJ": {
                 "long_name": "Vertical Infinite Bomb Jump",
-                "description": "By timing bombs precisely, the player can ride their blasts to gain an infinite amount of height.",
+                "description": "By timing bombs precisely, Samus can gan an arbitrary amount of height. ",
```
Comment (Miepee) [inline randovania/games/super_metroid/json_data/header.json:268]: *gain
Also please remove the whitespace between dot and quotation mark at the end

File: randovania/games/super_metroid/json_data/header.json [hunk 2]
```diff
@@ -323,19 +323,14 @@
...
             "HBJ": {
                 "long_name": "Horizontal Infinite Bomb Jump",
-                "description": "",
+                "description": "By timing bombs precisely, Samus can gain an arbitrary amount of horizontal distance without touching the ground.",
                 "extra": {}
             },
             "MidAirMorph": {
                 "long_name": "Mid-Air Morph",
-                "description": "",
+                "description": "Morphing while in the air allows Samus to get into elevated morph ball tunnels without needing spring ball or bombs.",
```
Comment (Miepee) [inline randovania/games/super_metroid/json_data/header.json:333]: *Morph Ball, capitalized please
same for Spring Ball and Bombs

Comment (Logarhythmic) [reply randovania/games/super_metroid/json_data/header.json:333]: I'll get into the habit of treating all items like this.

File: randovania/games/super_metroid/json_data/header.json (hunk 1 above)
Comment (Logarhythmic) [reply randovania/games/super_metroid/json_data/header.json:268]: Got the white space at the beginning of 'Gravity Jump too

**Review by Miepee — APPROVED** [review]

**Pull request merged**